hedera-node/hedera-smart-contract-service-impl/src/main/java/com/hedera/node/app/service/contract/impl/exec/TransactionProcessor.java

```diff
@@ -100,23 +97,18 @@ public HederaEvmTransactionResult processTransaction(
             @NonNull final HederaEvmContext context,
             @NonNull final ActionSidecarContentTracer tracer,
             @NonNull final Configuration config) {
-        final InvolvedParties parties;
-        final GasCharges gasCharges;
-        final MessageFrame initialFrame;
-        try {
-            parties = computeInvolvedParties(transaction, updater, config);
-            gasCharges = gasCharging.chargeForGas(parties.sender(), parties.relayer(), context, updater, transaction);
-            initialFrame = frameBuilder.buildInitialFrameWith(
-                    transaction,
-                    updater,
-                    context,
-                    config,
-                    parties.sender().getAddress(),
-                    parties.receiverAddress(),
-                    gasCharges.intrinsicGas());
-        } catch (HandleException failure) {
-            return HederaEvmTransactionResult.abortFor(failure.getStatus());
-        }
+        // Setup for the EVM transaction; thrown HandleException's will propagate back to the workflow
```

Comment:
I'm puzzled by the removal of the try catch.  It seems like some methods below can throw exceptions.  Is it handled by one of the callers?

Reply:
There's really no difference between a `HandleException` thrown here and one thrown [here](https://github.com/hashgraph/hedera-services/blob/07825-xtest-calls-and-storage/hedera-node/hedera-smart-contract-service-impl/src/main/java/com/hedera/node/app/service/contract/impl/exec/ContextTransactionProcessor.java#L82) just before entering the `TransactionProcessor`. (Both are pre-EVM failures/aborts.)

So I thought it was confusing and redundant to have an `abortReason` version of the `HederaEvmTransactionResult`.

Reply:
Oh, I see your reasoning.  Makes sense.